pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -84,15 +84,15 @@ sub _check_for_apache_chroot {
             {
                 'key'        => 'Apache_vhosts_not_segmented',
                 'type'       => $Cpanel::Security::Advisor::ADVISE_BAD,
-                'text'       => ['Apache vhosts are not segmented or chroot()ed.'],
-                'suggestion' => [
+                'text'       => $self->_lh->maketext('Apache vhosts are not segmented or chroot()ed.'),
+                'suggestion' => $self->_lh->maketext(
```

Comment:
Yes.  The problem with newlines only occurs when a newline exists within a string: e.g.

```perl
                q{MySQL test database is used by numerous attacks and should be removed.
  				> mysql -e 'drop database test'
  			}
```

However, this example gets fixed in a later commit.  The scope of this commit is only to move where the maketext call happens.

The output of `lt find . --deviants` (on the final commit) is `Nothing to report`.